Upstream: no

We can't ship specifically pyee 8.1.0, we're not a stable release distribution, so relax the requirements

diff --git a/setup.py b/setup.py
index 189a6bd..4b65f01 100644
--- a/setup.py
+++ b/setup.py
@@ -46,7 +46,7 @@ setup(
     ],
 
     install_requires=[
-        "pyee==8.1.0",
+        "pyee>=8.1.0",
         "six>=1.10.0"
     ]
 )
